keras/layers/core.py

```diff
@@ -46,6 +48,17 @@ def build(self):
         pass
 
     @property
+    def trainable(self):
```

Comment:
What is the justification for using a decorator over a boolean attribute `self.trainable` (set by default to True)?